backend/internal/api/api_test.go

```diff
@@ -573,4 +577,34 @@ func TestDBPostHandler(t *testing.T) {
 		assert.Equal(t, http.StatusOK, recorder.Code)
 
 	})
+
+	t.Run("should log proper error when sending event fails", func(t *testing.T) {
+		//given
+		hook := test.NewGlobal()
+		value := `{` +
+			`"id":"` + fixedID + `",` +
+			`"content":"base64",` +
+			`"gcp":"",` +
+			`"status":false` +
+			`}`
+
+		req, err := http.NewRequest("POST", "/v1/images", bytes.NewBuffer([]byte(value)))
+		assert.NoError(t, err)
+		req.Header.Set("Content-Type", "application/json")
+		recorder := httptest.NewRecorder()
+		dbManagerMock := apiMocks.DBManager{}
+		idMock := utilsMocks.IdGenerator{}
+		idMock.On("NewID").Return(fixedID, nil)
+		testSubject := NewHandler(&dbManagerMock, &idMock)
+		dbManagerMock.On("InsertToDB", fixedID, value).Return(nil)
+		eventHelperMock := eventMocks.EventHelper{}
```

Comment:
You define a mock object that is not used. I added the following line to the test:
`eventHelperMock.AssertExpectations(t)`

Tests failed as follows:
`
api_test.go:578: FAIL: 0 out of 1 expectation(s) were met.
        	The code you are testing needs to make 1 more call(s).
        	at: [api_test.go:578]
`
So the test is using the real implementation, not the mock.